src/platform/mhv/tests/hooks/useMyHealthAccessGuard.unit.spec.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { waitFor } from '@testing-library/react';
+
+import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import { useMyHealthAccessGuard } from '../../hooks/useMyHealthAccessGuard';
+
+const stateFn = ({ mhvAccountState = 'OK' } = {}) => ({
+  user: {
+    profile: {
+      mhvAccountState,
+    },
+  },
+});
+
+const Component = () => {
+  useMyHealthAccessGuard();
+  return <h1>Health Tool</h1>;
+};
+
+const setup = (initialState = stateFn()) =>
+  renderWithStoreAndRouter(<Component />, {
+    initialState,
+  });
+
+describe('useMyHealthAccessGuard', () => {
+  let originalLocation;
+
+  beforeEach(() => {
+    originalLocation = window.location;
+    delete window.location;
+    window.location = {
+      replace: sinon.spy(),
+    };
+  });
+
+  afterEach(() => {
+    window.location = originalLocation;
+  });
+
+  describe('with valid MHV account', () => {
```

Comment:
Why have a describe and it? Just have the it.

Reply:
The `describe` blocks detail the unit under test and the conditions. The 'it' blocks describe the outcome.